scripts/install_epinio.sh

```diff
@@ -37,6 +37,8 @@ helm upgrade --debug --wait --install -n epinio --create-namespace epinio helm-c
   --set global.domain=${EPINIO_SYSTEM_DOMAIN} \
   --set server.accessControlAllowOrigin="https://${MY_HOSTNAME}" \
   ${INSTALL_OPTIONS} \
+  --set global.tlsIssuer=letsencrypt-epinio \
+  --values script/values-users.yaml \
```

Comment:
When I run the test [here](https://github.com/epinio/epinio-end-to-end-tests/runs/7237292295?check_suite_focus=true) in Github apparently this is not the right path.
```
helm.go:88: [debug] open script/values-users.yaml: no such file or directory
```
I am not sure which one should be as I tried as well `values-users.yaml` solely and did not work either.
@juadk could please help me here? Thanks

Reply:
When the helm install command is triggered, you are in the epinio repo:
https://github.com/epinio/epinio-end-to-end-tests/blob/main/scripts/install_epinio.sh#L6
```
+ pushd epinio
~/actions-runner/_work/epinio-end-to-end-tests/epinio-end-to-end-tests/epinio ~/actions-runner/_work/epinio-end-to-end-tests/epinio-end-to-end-tests
```
The values file you are pushing is in `~/actions-runner/_work/epinio-end-to-end-tests/epinio-end-to-end-tests/scripts/`

Try with `../epinio-end-to-end-tests/scripts/values-users.yaml`

Reply:
Thanks @juadk  but apparently that one does not work either: 
https://github.com/epinio/epinio-end-to-end-tests/runs/7251558149?check_suite_focus=true
`helm.go:88: [debug] open ../epinio-end-to-end-tests/scripts/values-users.yaml: no such file or directory`

I tried to come up with the path but it was not clear. I will keep trying to figure it out

Reply:
oh try this one then:
```
../scripts/values-users.yaml
```
because the path is `'/home/sles/actions-runner/_work/epinio-end-to-end-tests/epinio-end-to-end-tests/epinio'`

Reply:
Looks like it works :smile:

Reply:
yes, thank your Enrico showed me that one right away and worked.
Let's see now the issue verifying the credentials